project/Dependencies.scala

```diff
@@ -498,13 +510,35 @@ object Dependencies {
     "io.netty" % "netty-handler" % NettyVersion,
     "io.netty" % "netty-transport" % NettyVersion,
 
-    "org.apache.cassandra" % "cassandra-all" % CassandraAllVersion % Test exclude("io.netty", "netty-all"),
+    "org.apache.cassandra" % "cassandra-all" % CassandraAllVersion excludeAll(
+      ExclusionRule("io.netty", "netty-all"),
+      ExclusionRule("ch.qos.logback", "logback-core"),
+      ExclusionRule("ch.qos.logback", "logback-classic")
+    ),
+
     "io.netty" % "netty-codec-http" % NettyVersion % Test,
     "io.netty" % "netty-transport-native-epoll" % NettyVersion % Test classifier "linux-x86_64",
     "com.fasterxml" % "classmate" % "1.3.0" % Test,
-    "org.apache.httpcomponents" % "httpclient" % "4.5.2" % Test
+    "org.apache.httpcomponents" % "httpclient" % "4.5.2",
+
+
+    // These deps are depended on by cassandra-all, and need to be upgraded in order to be consistent with transitive
+    // dependencies from our other libraries
+    "com.lmax" % "disruptor" % "3.3.6",
+    "javax.validation" % "validation-api" % "1.1.0.Final",
+    "org.hibernate" % "hibernate-validator" % "5.2.4.Final",
+    jbossLogging,
+    "com.fasterxml" % "classmate" % "1.3.0",
+    "org.slf4j" % "log4j-over-slf4j" % "1.7.21",
+    "org.xerial.snappy" % "snappy-java" % "1.1.2.6",
+    "org.yaml" % "snakeyaml" % "1.16",
+
+    "org.apache.httpcomponents" % "httpcore" % "4.4.4",
+    "commons-logging" % "commons-logging" % "1.2",
+    "junit" % "junit" % "4.11"
```

Comment:
This _is_ the ripple effect. For some reason, when excluding `logback`  all this dependencies required manual intervention.